### DISCUSSION TITLE
left and top = null when i add text

### DISCUSSION THREAD

**Discussion by khaledmndev** [fabricjs/fabric.js#8170] [Q&A]
## Version
5.2.1

## Test Case
- [ ] [Browser Issue Template](https://jsfiddle.net/Lcp2h3nv/)

## Information about environment
Nodejs or browser? Browser
Which browsers? Edge 104.0.1293.63 (Official build) (64-bit)

## Steps To Reproduce

1. for short: i can add custom text on computer on cursor position but not on touch devices like mobile or tablet

## Expected Behavior
Here you can see on computer it's working hen i add text
![alt text](https://i.ibb.co/PwKRMHr/Screenshot-2022-08-21-184627.png)

## Actual Behavior
And here's what happens when i try to add on mobile or tablet:
![alt text](https://i.ibb.co/gzyLjc2/Screenshot-2022-08-21-185006.png)

**Comment by ShaMan123** [comment]
I closed your issue because you didn't follow the guidelines and issue template...
Opening the same ticket under a discussion is simply **SPAMMING**

Fill out the issue template from top to bottom.
If you don't understand something call out.

NO reproduction = waste of time

**Reply by ShaMan123** [reply]
If it is a bug we will look in to it. But there is no way we can with what you wrote

**Reply by khaledmndev** [reply]
hey, thanks for replying to me,  i actually didn't understand what is missing?!!
do you want a live demo of it?

**Reply by ShaMan123** [reply]
yes
a live concise demo is a must
